### PR TITLE
feat: Added support for Playwright

### PR DESCRIPTION
**Pull request by TriPSs**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,6 +39,7 @@
   },
   "homepage": "https://github.com/moshensky/pdf-visual-diff#readme",
   "dependencies": {
+    "@playwright/test": "^1.40.1",
```
Comment (moshensky) [inline package.json:42]: 👋 

Thank you for your contribution. 
Are there any options not to depend directly on playwrigit?

Comment (TriPSs) [reply package.json:42]: Ow, this one should be in the dev dependencies

**Comment by ComaToastUK** [comment]
Hi @moshensky, this would actually be really useful, any chance this will be merged?

**Comment by moshensky** [comment]
> Hi @moshensky, this would actually be really useful, any chance this will be merged?

I'll have a look tomorrow.

**Comment by TriPSs** [comment]
@ComaToastUK FYI: I created [to-match-pdf-snapshot](https://github.com/Doczilla-APP/to-match-pdf-snapshot) which is specifically for Playwright

**Pull request closed without merging**